Packet++/src/SmtpLayer.cpp

```diff
@@ -19,6 +19,12 @@ namespace pcpp
 
 		for (size_t idx = 0; idx < field.size(); ++idx)
 		{
+			// Truncate the value if it is longer than 8 bytes
+			if (idx >= 8)
```

Comment:
Can we not use std::min between field.size and 8?

Reply:
Good idea!